Meta/WPT.sh

```diff
@@ -122,6 +122,12 @@ done
 WPT_ARGS+=( "--binary=${LADYBIRD_BINARY}" )
 TEST_LIST=( "$@" )
 
+for i in "${!TEST_LIST[@]}"; do
+    item="${TEST_LIST[i]}"
+    item="${item#*Tests/LibWeb/WPT/wpt/}"
```

Comment:
This may not be the correct path if `WPT_SOURCE_DIR` is set to something other than the default value.

Reply:
> This may not be the correct path if `WPT_SOURCE_DIR` is set to something other than the default value.

aha — OK, I’ll try to figure out if I can make it handle that

Reply:
Please take another look now